jib-cli/src/test/java/com/google/cloud/tools/jib/cli/war/WarFilesTest.java

```diff
@@ -0,0 +1,75 @@
+/*
+ * Copyright 2021 Google LLC.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.google.cloud.tools.jib.cli.war;
+
+import static com.google.common.truth.Truth.assertThat;
+import static org.mockito.Mockito.when;
+
+import com.google.cloud.tools.jib.api.InvalidImageReferenceException;
+import com.google.cloud.tools.jib.api.JibContainerBuilder;
+import com.google.cloud.tools.jib.api.buildplan.AbsoluteUnixPath;
+import com.google.cloud.tools.jib.api.buildplan.ContainerBuildPlan;
+import com.google.cloud.tools.jib.api.buildplan.FileEntriesLayer;
+import com.google.common.collect.ImmutableList;
+import java.io.IOException;
+import java.nio.file.Paths;
+import java.util.Arrays;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+/** Tests for {@link WarFiles}. */
+@RunWith(MockitoJUnitRunner.class)
+public class WarFilesTest {
+
+  @Mock private StandardWarExplodedProcessor mockStandardWarExplodedProcessor;
+
+  @Test
+  public void testToJibContainerBuilder_explodedStandard_basicInfo()
+      throws IOException, InvalidImageReferenceException {
+    FileEntriesLayer layer =
+        FileEntriesLayer.builder()
+            .setName("classes")
+            .addEntry(
+                Paths.get("path/to/tempDirectory/class1.class"),
+                AbsoluteUnixPath.get("/my/app/class1.class"))
+            .build();
+    when(mockStandardWarExplodedProcessor.createLayers()).thenReturn(Arrays.asList(layer));
+    when(mockStandardWarExplodedProcessor.computeEntrypoint(ArgumentMatchers.anyList()))
+        .thenReturn(ImmutableList.of("java", "-jar", "/usr/local/jetty/start.jar"));
+
+    JibContainerBuilder containerBuilder =
+        WarFiles.toJibContainerBuilder(mockStandardWarExplodedProcessor);
+    ContainerBuildPlan buildPlan = containerBuilder.toContainerBuildPlan();
+
+    assertThat(buildPlan.getBaseImage()).isEqualTo("jetty");
+    assertThat(buildPlan.getEntrypoint())
+        .isEqualTo(ImmutableList.of("java", "-jar", "/usr/local/jetty/start.jar"));
```

Comment:
Why not `.containersExactly(...).inOrder()`?

Reply:
Done